### PR TITLE
[DOCS-7139] Alfresco Enterprise Viewer 3.5.1

### PR DESCRIPTION
**Pull request by Amy-Hyland**
Add ACS 7.4 support

### CODE REVIEW

File: enterprise-viewer/latest/support/index.md [hunk 1]
```diff
@@ -2,10 +2,11 @@
 title: Supported platforms
 ---
 
-The following are the supported platforms for the Alfresco Enterprise Viewer 3.5:
+The following are the supported platforms for the Alfresco Enterprise Viewer 3.5.1:
```
Comment (anxumalo) [inline enterprise-viewer/latest/support/index.md:5]: Let's keep the version as either `3.5` or `3.5.x` to cover the released versions, `3.5.0` and `3.5.1`.
```suggestion
The following are the supported platforms for the Alfresco Enterprise Viewer 3.5:
```

**Review by anxumalo — COMMENTED** [review]
@Amy-Hyland  I've added a few suggestions - see what you think

**Review by anxumalo — DISMISSED** [review]
LGTM

**Review by katiebrill — COMMENTED** [review]
Perfect. Thank you!

**Comment by Amy-Hyland** [comment]
This should wait to be merged when https://github.com/Alfresco/docs-alfresco/pull/1030 is approved and merged in.

**Pull request merged**